beacon/goclient/goclient.go

```diff
@@ -101,13 +103,24 @@ type MultiClient interface {
 	eth2client.BeaconBlockRootProvider
 	eth2client.SyncCommitteeContributionProvider
 	eth2client.SyncCommitteeContributionsSubmitter
+	eth2client.BeaconBlockHeadersProvider
 	eth2client.ValidatorsProvider
 	eth2client.ProposalPreparationsSubmitter
 	eth2client.EventsProvider
 	eth2client.ValidatorRegistrationsSubmitter
 	eth2client.VoluntaryExitSubmitter
 }
 
+type operatorDataStore interface {
+	AwaitOperatorID() spectypes.OperatorID
+}
+
+type EventTopic string
+
+const (
+	HeadEventTopic EventTopic = "head"
```

Comment:
to follow Go enum standards:
```suggestion
	TopicHeadEvent EventTopic = "head"
```

Reply:
pushed the change. Do not commit the suggestion because all references needs to be updated as well